### PR TITLE
WIP: Jordan/1625 sentry source maps

### LINKED ISSUE #1625: Set release and upload sourcemaps to Sentry on release
To use the errors properly we need to map the users Voyager to a release on Sentry and provide that release with the source maps so Sentry can give us a propper stack trace.

### PR DESCRIPTION
**Pull request by jbibla**
Closes #1625 

_Description:_

- installed and configured sentry CLI
- installed webpack plugin for sourcemaps

❤️ Thank you!

---

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

**Comment by jbibla** [comment]
this will successfully upload source maps as artifacts per version (commit hash) ... however, the errors that sentry reports are still failing to actually _use_ the sourcemap. 

i could use some help figuring this out. it may be working as is but i have not been able to successfully build voyager with this code included and cannot confirm.

File: app/package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (faboweb) [inline app/package.json]: This file should be ignored, it is created while building Voyager

File: webpack.renderer.config.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const fs = require(`fs-extra`)
 
 const HtmlWebpackPlugin = require(`html-webpack-plugin`)
 const VueLoaderPlugin = require(`vue-loader/lib/plugin`)
+const SentryWebpackPlugin = require(`@sentry/webpack-plugin`)
```
Comment (faboweb) [inline webpack.renderer.config.js]: nice

File: .sentryclirc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[defaults]
```
Comment (faboweb) [inline .sentryclirc]: should this file be checked in?

**Comment by faboweb** [comment]
Added some more stuff. Easy to test: `node tasks/build/build.js --commit={your pushed commit} --network=local-testnet` > run the application locally.

**Comment by jbibla** [comment]
closing for now. will re-open after results from next release.

**Pull request closed without merging**